tests/manage/mcg/ui/test_mcg_ui.py

```diff
@@ -289,6 +299,6 @@ def test_obc_creation_and_deletion(self, setup_ui_class, storageclass, bucketcla
         ), f"BucketClass mismatch. Expected: {bucketclass}, found: {obc_bucketclass}"
 
         logger.info(f"Delete {obc_name}")
-        obc_ui_obj.delete_obc_ui(obc_name)
+        obc_ui_obj.delete_obc_ui(obc_name, delete_via)
```

Comment:
As per the BZ 2097772, we should try deleting the  Object Bucket which should not be allowed and when we hover over the 3 dots we should see a Message 'The corresponding ObjectBucketClaim must be deleted first.'
where is it covered? Am I missing something?

Reply:
As we discussed in private, this PR will be a basis to cover BZ #2097772 and it still good to merge it, having a happy path scenario valid for 4.12
Thank you